galaxy/api/internal/views/combined.py

```diff
@@ -109,3 +109,10 @@ def get(self, request):
         }
 
         return response.Response(result)
+
+    def _validate_order(self, order):
+        allowed_orders = ['download_count', 'name', '-download_count', '-name']
+        if order not in allowed_orders:
+            raise InvalidParams(
+                detail="Order must be one of " + str(allowed_orders)
```

Comment:
noticed sometimes the use of `detail=` vs `details=` in `InvalidParams()`.

also very minor... for future consistency, we usually use `%` or `.format()` for string concatenation (and I guess the `+` spacing looks a bit odd in an argument)

Reply:
According to documentation on the base exception class `APIException`, the correct parameter is `detail` so the other uses are incorrect and may well cause errors when they are hit. Two notes:

1. @newswangerd Let's make sure the original reviewers sign off on a PR before merging it when it has specifically been marked with requests. A little delay can save us some pain and breakage.
2. The lines using `details` will actually cause an error of their own (exceptions causing exceptions! the humanity of it all!) but all the tests are passing. That probably means there are some gaps in the tests we could fill in by making sure these failure cases are covered, not just the successful cases.

https://www.django-rest-framework.org/api-guide/exceptions/#apiexception